ca.potatoe.cmdjewel.yml

```diff
@@ -0,0 +1,64 @@
+app-id: ca.potatoe.cmdjewel
```

Comment:
how do I kno you own https://potatoe.ca/? Can you put something up like a github link etc,?

Reply:
Go to https://potatoe.ca/ -> click "me!" in the header -> github linked